kolibri/core/lessons/serializers.py

```diff
@@ -128,7 +132,29 @@ def update(self, instance, validated_data):
                 lesson_id=instance.id,
                 collection_id__in=(set(current_group_ids) - set(new_group_ids))
             ).delete()
-
+        if validated_data.get("is_active", False):
```

Comment:
The other problem here is that toggling from 'active' to 'inactive' and then back to 'active' will clear any learner's engagement with the content also. Hrm.